nginx.yaml

```diff
@@ -138,8 +138,6 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/usr/lib/nginx/modules
           mkdir -p ${{targets.subpkgdir}}/etc/nginx/modules
 
-          mv ${{targets.destdir}}/ngx_${{range.key}}_module.so ${{targets.subpkgdir}}/usr/lib/nginx/modules/
```

Comment:
does this `mv` need to be removed?

Reply:
yes, because in the compilation the modules are generated in `/usr/lib/nginx/modules`

Reply:
you are right josh, the mv changes but it's needed. I've push the change.